src/OpenSpaceNet.cpp

```diff
@@ -276,6 +281,7 @@ void OpenSpaceNet::initFeatureSet()
     VectorOpenMode openMode = args_.append ? APPEND : OVERWRITE;
 
     featureSet_ = make_unique<FeatureSet>(args_.outputPath, args_.outputFormat, args_.layerName, definitions, openMode);
+    featureSet_->setProjection(sr_.wkt());
```

Comment:
We did but I kept the interface the same to skirt an issue:

FeatureSet has a member ds_ whose member function CreateLayer which requires an argument of type OGRSpatialReference.  Currently there is no way to access the underlying OGRSpatialReference in the SpatialReference class and it seems weird to provide a getter for it since it is the only member variable of SpatialReference.  Access to the underlying OGRSpatialReference can be added by either providing a getter in SpatialReference, specifying FeatureSet as a friend of SpatialReference, or making SpatialReference a stuct instead of a class.

I'm not sure which way you would prefer.